public/index.html

```diff
@@ -21,6 +22,10 @@
 
 <div class="ractivef hide">
 
+    <uxrow>
+        <uxblah></uxblah>
+    </uxrow>
```

Comment:
Do we need this in markup as there's no such component yet ? Or it's just for demo?